openedx/stanford/djangoapps/auth_lagunita/management/commands/backfill_auth_lagunita_extrainfo.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""
+Backfill extra registration info for existing users
```

Comment:
Why is the backfill required?

Reply:
1. I cargo-culted from the CME registration.

2. I can't completely remember now why we did that, but I seem to recall possibly running in to an error when users didn't have `Info`/`ExtraInfo`?

But maybe that's just how we wrote the data report logic?
Can you think of any reason why we'd need this object?

As is, our management command will only be looking up `Info` objects and then acting on the attached user, not looking up users and then manipulating their `Info`.

So maybe this is just an unnecessary cargo-cult that is no longer needed, since we're also not porting over existing data, like we had done for CME?

Any other thoughts on that?

Reply:
> I can't completely remember now why we did that, but I seem to recall possibly running in to an error when users didn't have `Info`/`ExtraInfo`?

I went ahead and tested a few things locally...

> But maybe that's just how we wrote the data report logic?

> As is, our management command will only be looking up `Info` objects and then acting on the attached user, not looking up users and then manipulating their `Info`.

Yeah, this is the crux of it; if we backfill data, you can safely reference the `info` object directly from the `User` model via `user.info`.

Without it, you need to check for existence:

```py
if hasattr(user, 'info'):
    pass
```

or provide a fallback default:

```py
info = getattr(user, 'info', Info())
```

or perform a separate lookup:

```py
try:
    info = Info.objects.get(user_id=user.id)
except Info.DoesNotExist:
    info = Info()
```

or only perform lookups via `Info` models, not `User`:

```py
infos = Info.objects.all()
```

One other thing to check would be what happens with Direct Access users.

I'll go ahead and remove the explicit ID/primary key assignment that you pointed out elsewhere, and then I'm inclined to merge this as is. I'll keep looking into how DA users are affected and we can use those findings to determine whether or not we actually run the backfill operation (which we can decide on up until we go live in prod). If we end up not running the command, I'll remove it in a follow-up PR.

Thoughts @dcadams ? 
CC: @kluo

Reply:
@dcadams @kluo I confirmed that direct access users do _not_ receive an `Info` object upon creation of the user model, so we can't rely on that connection.

Given that, we'll need all code to be resilient to missing `info` anyway,
so I don't think we gain much by keeping this.
I'm fine to drop the backfill command

Final thoughts?

Reply:
Drop it.